beetles.py

```diff
@@ -54,6 +56,10 @@ def __init__(self, team, center_x=0, center_y=0):
         self.known_enemies = None
         self.active = False
         self.spatial_manager = None
+        self.sfx_beetle_dead = arcade.load_sound(SFXPATH_BEETLE_DEAD)
+
+        # Commenting out because we will need hit/damage SFX but dunno where that goes yet
+        # self.sfx_beetle_hit = arcade.load_sound(BEETLE_SFXPATH_BEETLE_HIT)
```

Comment:
I guess it would have to go on the collision detection! This might be more reason to set up a sound manager...